roles/common/files/mirrorlist

```diff
@@ -0,0 +1,572 @@
+##
```

Comment:
Does it make sense to rank the servers using `rankmirrors` from `pacman-contrib` ?

Reply:
Sure but that should run on the target on a timer, feel free to implement.